src/step1-index.js

```diff
@@ -0,0 +1,57 @@
+import {getDividedInteger} from './utils/getDividedInteger';
+import {queryValidInput} from './view/queryValidInput';
+import {Lotto} from './domain/Lotto';
+import {splitNumbers} from './utils/splitNumbers';
+import {PRIZE_MAP} from './domain/constants/prizeMap';
+import {formatPrizeKR} from './domain/formatPrizeKR';
+import {getPercentage} from './utils/getPercentage';
+import {validateOnlyNumber} from './utils/validateOnlyNumber';
+import {getLottoResult} from './domain/getLottoResult';
+import {getTotalBenefit} from './domain/getTotalBenefit';
+import {validateWinningNumbers} from './domain/validateWinningNumbers';
+import {validateBonusNumber} from './domain/validateBonusNumber';
+import {validateRetryType} from './domain/validateRetryType';
```

Comment:
```suggestion
import {
  getDividedInteger
  splitNumbers,
  getPercentage,
  validateOnlyNumber,
} from './utils';
import {
  Lotto,
  PRIZE_MAP,
  formatPrizeKR,
  getLottoResult,
  getTotalBenefit,
  validateWinningNumbers,
  validateBonusNumber,
  validateRetryType,
} from './domain';
import {queryValidInput} from './view';
```
이렇게 쓰일 수 있도록 index.js를 하나 만들어서 관리해주면 어떨까요?
이럴 경우 의존관계가 명확하게 구분되어서 코드를 읽고 해석할 때 수월하답니다!

Reply:
단순히 가독성 개선이라고만 생각했는데 의존관계를 구분하는 역할을 하는군요!